src/component_managers/astt_comp_manager.py

```diff
@@ -391,33 +346,25 @@ def track_sun(self, duration_time, az_speed=None, el_speed=None):
         begin_az = track_start_position.azimuth
         begin_el = track_start_position.elevation
         while count < time_conversion and not self.trackstop:
-            track_time = datetime.datetime.now(
-                datetime.timezone.utc
-            ) + datetime.timedelta(seconds=10)
+            track_time = datetime.datetime.now(datetime.timezone.utc) + datetime.timedelta(
+                seconds=10
+            )
             future_timestamp = datetime.datetime.timestamp(track_time)
             dt = future_timestamp - begin_time
             if az_speed is not None and el_speed is not None:
                 az_desired = begin_az + (az_speed * dt)
                 el_desired = begin_el + (el_speed * dt)
             else:
                 az_desired, el_desired = sun.get_sun_az_el(track_time)
-            self.point_to_coordinates(
-                future_timestamp, az=az_desired, el=el_desired
-            )
+            self.point_to_coordinates(future_timestamp, az=az_desired, el=el_desired)
             if not self.transmission_triggered:
                 self.trigger_transmission()
             else:
                 pass
             time.sleep(5)
-            self.logger.info(
-                "---------------------------------------"
-            )
-            self.logger.info(
-                f"Sun_Az : {az_desired}, Sun_El : {el_desired}"
-            )
-            self.logger.info(
-                "---------------------------------------"
-            )
+            self.logger.info("---------------------------------------")
+            self.logger.info(f"Sun_Az : {az_desired}, Sun_El : {el_desired}")
+            self.logger.info("---------------------------------------")
```

Comment:
It should be removed in the future

Reply:
We will have to work it on the decoupling ticket. I propose we setup a meeting with a Meerkat guy to understand how they give out info from an operation say slew, track or point.